tests/triggering/fam-finish-any/suite.rc

```diff
@@ -3,11 +3,11 @@
         R1 = """FAM:finish-any => foo"""
 [runtime]
     [[FAM]]
-        script = "sleep 10"
+        script = sleep 10
```

Comment:
Why did this one slip the net?

Reply:
This one is very hard to change, and I ran out of time on Thursday.